### DISCUSSION TITLE
About finder in Oculars plugin.

### DISCUSSION THREAD

**Discussion by 3domennotry** [Stellarium/stellarium#3822] [General]
Hello.
(Sorry, I can not speak English very well.)

I have 5cm finder.

I set the parameters like binoculars.
![20240724_1](https://github.com/user-attachments/assets/b38f06f9-6aa9-4983-8a06-93add4f51f7f)

However, horizontal and vertical are not reversed.
(My viewfinder has the image inverted.)

How do you all set it up?

thank you.

**Comment by gzotti** [comment]
If you declare your finder as 'Binoculars', the view is always upright like in binoculars. Configure it as telescope.

**Comment by 3domennotry** [comment]
Thank you for reply.

I tried two methods.

A. Define telescope and eye piece.

B. Define binocular. Horizontal and Vertical flip button click.

I used each of these methods and the results were the same.

thank you.

**Comment by 3domennotry** [comment]
I close this discussion.
thank you.
